test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -133,11 +144,12 @@ public void actionPerformed(ActionEvent event) {
             public void actionPerformed(ActionEvent event) {
                 try {
                     Image image;
-                    java.util.List<Image> list = new java.util.ArrayList();
+                    List<Image> list = new ArrayList<>();
                     for (int i = 1; i <= 4; i++) {
                         String fileName = fileBase + "T" + i + ".gif";
                         image = Toolkit.getDefaultToolkit().getImage(fileName);
-                        PassFailJFrame.log("Loaded image " + fileName + ". setting to the list for frame");
+                        PassFailJFrame.log("Loaded image " + "T" + i + ".gif."
+                                           + " Setting to the list for frame");
```

Comment:
Why can't `fileName` be used in the logging?

```suggestion
                        PassFailJFrame.log("Loaded image " + fileName + "."
                                           + " Setting to the list for frame");
```